xbmc/platform/win32/WIN32Util.h

```diff
@@ -27,6 +27,7 @@
 #include "MediaSource.h"
 #include "guilib/Geometry.h"
 #include "powermanagement/PowerManager.h"
+#include "ServiceBroker.h"
```

Comment:
This shouldn't be needed

Reply:
thanks, seems my script didn't work 100%